app/src/forms/form/service.js

```diff
@@ -17,10 +18,16 @@ const service = {
     try {
       trx = await transaction.start(Form.knex());
 
-      const obj = Object.assign({}, data);
+      const obj = {};
       obj.id = uuidv4();
+      obj.name = data.name;
+      obj.description = data.description;
+      obj.shortName = data.shortName ? data.shortName : obj.id.substring(0, 8).toUpperCase();
```

Comment:
Will need to discuss with team - but should decide if we still need a "shortname GUID" at all or not (depends on how we're thinking about handling form/submission version identification either by "autogen names, or something else".

In comfort, we did have short GUID confirmations, but there was never a direct mechanism to do lookup based on only that value - it was always only a derived value from the id key.

Reply:
I don't think it's quite as complicated as you are making it out to be.  But I am removing shortName for now and adding a backlog ticket so we don't waste time discussing this right now.